### DISCUSSION TITLE
Appreciation Post

### DISCUSSION THREAD

**Discussion by angelhdzmultimedia** [WarmUpTill/SceneSwitcher#529] [General]
This plugin is amazing! It's like a program inside a program.
Thank you to all people who made this possible.
I hope learning it more in depth, as I started automatizing things recently.

**Comment by WarmUpTill** [comment]
Thank you very much for the kind words :)
If you have questions do not hesitate to ask or have a look at the [wiki](https://github.com/WarmUpTill/SceneSwitcher/wiki).
